pkg/rest/list.go

```diff
@@ -28,8 +28,16 @@ import (
 
 const emptyResponseRetryMax = 10
 
+// AddEntriesToResult is a function which should parse an API response body and append the returned entries to a result slice
+// Handling the parsing, any possible filtering and owning and filling the result list is left to the caller of ListPaginated,
+// as it might differ notably between client implementations.
+// The function MUST return the number of entires it has parsed from the received API payload body. This is used to validate
```

Comment:
```suggestion
// The function MUST return the number of entries it has parsed from the received API payload body. This is used to validate
```

Reply:
Thanks, correct on the commit that introduced it.